ha/core/controllers/pcs/node_controller.py

```diff
@@ -367,7 +368,15 @@ def stop(self, node_id: str, timeout: int = -1, **op_kwargs) -> dict:
                 # Stop services on node except sspl-ll
                 self._controllers[const.SERVICE_CONTROLLER].stop(node_id=node_name, excludeResourceList=[RESOURCE.SSPL_LL.value])
 
-                # TODO: storage enclosure is stopped on the node
+                # Stop the storage enclosure on the node
+                actuator_mgr = ActuatorManager()
+                encl_status = actuator_mgr.enclosure_stop(node_name)
+                if encl_status == False:
+                    Log.warn(f"Enclosure stop failed for {node_name}")
```

Comment:
We should return from here with error response if the enclosure stop fails.

Reply:
Also, I guess this is not the right place to power-off the node. We should do it before updating node health and node power off.
Also, we need to update the enclosure status in the system health.

Reply:
Actually an exception will be raised if the enclosure stop failed. 
So we can remove the check and the  Log.warn

Reply:
Regarding 

> Also, we need to update the enclosure status in the system health
TODO comment was  already there in the node stop PR for this 

> Also, I guess this is not the right place to power-off the node. We should do it before updating node health and node power off.
Once node health is updated (TODO) doing poweroff here will be fine right ?

Reply:
>We should return from here with error response if the enclosure stop fails.
Removed the check and the warning

Reply:
"TODO comment was already there in the node stop PR for this"
Which task this comment will be handled?

Reply:
"Once node health is updated (TODO) doing poweroff here will be fine right ?"
Currently I see it is done before moving the node to standby, I see the services are stopped before this. Can we move enclosure power off after node standby?